src/pages/Home/Home.tsx

```diff
@@ -1,10 +1,26 @@
 import React from 'react';
 import SignInForm from '../../organisms/SignInForm/SignInForm';
+import SignUpForm from '../../organisms/SignUpForm/SignUpForm';
```

Comment:
with the exception of the css can you change the imports to absolut paths eg 
'organisms/ SignInForm/SignInForm';